docs/sphinx/themes/globalbftoc.html

```diff
@@ -1,4 +1,5 @@
 <h3><a href="{{ pathto(master_doc) }}">{{ _('Bio-Formats') }}</a></h3>
 {{ toctree()}}
-<a href="http://downloads.openmicroscopy.org/latest/bio-formats5.1/">{{ _('Downloads') }}</a></br>
+<a href="http://downloads.openmicroscopy.org/latest/bio-formats5.1/">{{ _('Main Downloads') }}</a></br>
+<a href="http://downloads.openmicroscopy.org/latest/bio-formats-cpp5.1/">{{ _('Bio-Formats C++ Downloads') }}</a></br>
```

Comment:
Since this is already in a "Bio-Formats" section, is having "Bio-Formats C++ Downloads" necessary (it will use precious horizontal space which could otherwise be used by the main content), or could it just be "C++ Downloads"?